test/Core/Utility/TestFramework.re

```diff
@@ -1,7 +1,84 @@
+open Rely.MatcherTypes;
+
+open Kernel;
+
+// TREE EXTENSIONS
+
+type treeExtensions = {toEqual: KeyedStringTree.t(int) => unit};
+
+let treeExtensions = (actual, {createMatcher}) => {
```

Comment:
Cool, I always forget about extending the matchers... this is a great in-depth example